frontend/app/components/UI/Button.tsx

```diff
@@ -10,13 +19,13 @@ const Button = ({ label, variant = "filled", ...rest }: TextButton) => {
 
   return (
     <button
-      aria-label={label}
```

Comment:
Trur jeg savner denne litt. Sida endringene her nå ikke sender inn en string, men en ReactNode, som innhold til knappen, så er det ingen garanti for at det oppfyller accessibility. Sida det kun er et hobbyprosjekt er dette kanskje ikke et smertepunkt, men tenker jo det er en bra vane å ha UU i ryggmargen når man koder frontend; også for moro.

Reply:
Jeg syntes dette er en bedre løsning UU-messig .. Man skal ikke trenge aria-label på en button så lenge den ellers er kodet semantisk korrekt, for iconButton derimot er jeg enig i at vi må ha med en aria-label, men jeg syntes ikke at man skal legge aria-labels på alle knapper sånn i tilfelle, burde vel heller prøve å skrive semantisk korrekt html der vi kan 😄

Reply:
Hvis vi trenger aria-label på knappen er det jo bare å legge det på der det trengs? Det eneste endringen min gjør er å ikke legge det på alle steder knappen brukes .. Men jeg ser ikke helt i hvilke tilfeller denne knappen kan bli brukt der det ville vært nødvendig? 🤔  Semantisk korrekt html var kanskje ikke riktig måte å forklare det på her, men poenget mitt var mer overordnet at man ofte ikke trenger aria-labels så lenge html-en er korrekt, det er faktisk frarådet UU-messig å overdrive bruken av aria-attributter der det ikke skal være 😄